build.fsx

```diff
@@ -107,6 +107,15 @@ Target "BuildApp" (fun _ ->
         |> DoNothing
 )
 
+Target "BuildDotNetCore" (fun _ ->
+    Fake.DotNetCli.Restore (fun p ->
+      {p with
+          WorkingDir = "./Octokit.Next" })
+
+    !! "./Octokit.Next/project.json"
```

Comment:
since `Fake.DotNetCli.Build` accepts a `seq<string>`, I went this way, taking example on other tasks. I don't know if it's right to do it this way since there's no globbing involved